package.yaml

```diff
@@ -88,6 +89,8 @@ library:
   - recursion-schemes >= 5.2.2.3
   - regex-base
   - regex-pcre-builtin
+  - smtlib-backends
+  - smtlib-backends-process
```

Comment:
do we not want to use `smtlib-backends-z3` here?

Reply:
I kept it general but we can have a go with the z3-specific backend, too.
Maybe we do that as a separate experiment?